browser/src/layer/tile/TileLayer.TableOverlay.js

```diff
@@ -324,6 +333,47 @@ L.CanvasTileLayer.include({
 		e.target.dragging.freezeY(true);
 	},
 
+	_addAddMarkers: function(type, start, end) {
+		const delta1 = this._convertPixelToTwips(this._selectionHeaderDistanceFromTable);
+		const markerSize = 40;
+		const markerSizeTwips = this._convertPixelToTwips(markerSize);
+		
+		let position, iconSize;
+		
+		if (type === 'column') {
+			const tableRightEdge = parseInt(this._currentTableData.columns.right);
+			const columnPosition = this._tablePositionColumnOffset + tableRightEdge + delta1 + markerSizeTwips/2;
+			const centerY = start + (end - start) / 2;
+			position = this._twipsToLatLng(new L.Point(columnPosition, centerY), this._map.getZoom());
+			iconSize = [20, 40];
+		} else {
+			const tableBottomEdge = parseInt(this._currentTableData.rows.right);
+			const rowPosition = this._tablePositionRowOffset + tableBottomEdge + delta1 + markerSizeTwips/2;
+			const centerX = start + (end - start) / 2;
+			position = this._twipsToLatLng(new L.Point(centerX, rowPosition), this._map.getZoom());
+			iconSize = [40, 20];
+		}
+		
+		const marker = L.marker(position, {
```

Comment:
We try to remove Leaflet library, please do not add more of it.
Or did you just copy existing example?

@gokaysatir might know how to replace

Reply:
I think we can use a section here. We are removing leaflet library. There are a few usages left. Now our infrastructure is ready for any drawing to the screen, i think we can add new features without latlng and related usage.

An HTMLObjectSection instance here should work. There are sections like "CursorHeaderSection" that extends the HTMLObjectSection. Those can be code pointers for this feature. When / if they are not enough, please inform me.

Thanks :)

Reply:
Changed from using leaflet library and created a new TableInsertMarkerSection